src/OneExplorer/OneExplorer.ts

```diff
@@ -1026,8 +1032,69 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<Node> {
   }
 
   /**
-   * Create ONE configuration file for a base model
-   * Input box is prefilled as <base model's name>.cfg
+   * Select the option for the configuration you want to create
+   * Return information about the selected option
+   * Default option is information about .cfg
+   *
+   * @param modelName A base model's name
+   * @param extName A base model's extension name
+   *
+   */
+  private async generateCfgInfo(
+    modelName: string,
+    extName: string
+  ): Promise<CfgInfo | undefined> {
+    const options = [
+      { label: ".cfg", description: "configuration file of onecc compiler" },
+    ];
+
+    if (extName === "tflite") {
+      options.push({
+        label: ".edgetpucfg",
+        description: "configuration file of edge tpu compiler",
+      });
+    }
+
+    const selectedOption = await vscode.window.showQuickPick(options, {
+      title: "Pick a configuration to create",
+      placeHolder: "cfg / edgetpucfg",
+    });
```

Comment:
Actually, this lines should regard the extName (.tflite / .onnx), so we need some conditional approches.

Reply:
I've completed this request and modularization additionally! :-)